be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidator.scala

```diff
@@ -85,17 +130,17 @@ object ElectionValidator extends MessageDataContentValidator with EventValidator
 
   def validateResultElection(rpcMessage: JsonRpcRequest): GraphMessage = {
```

Comment:
I thought that for now, when the server receives a resultElection it just answer with an error, so the validator could just return Right(PipelineError(ErrorCodes.SERVER_ERROR.id, "not implemented yet !", rpcMessage.getId))

Reply:
You say that, because we do not recieve any ResultElection message comming from the front end, right? So, no specifical test to do?